utils/utils.go

```diff
@@ -373,3 +373,21 @@ func Sync(path string) error {
 	}
 	return nil
 }
+
+// RemoveDupStorageOpts removes duplicated storage option from the list
```

Comment:
nit: I'm personally not a fan of putting functions in a utils package that isn't required in multiple packages. To me, utils should be used as a place to prevent cyclical dependencies. This function seems better suited for the config package IMO